k8s-path-cleanup.tf

```diff
@@ -38,6 +38,9 @@ resource "kubernetes_deployment_v1" "beekeeper_path_cleanup" {
       metadata {
         labels = local.path_cleanup_label_name_instance
         annotations = {
+          "ad.datadoghq.com/${local.path_cleanup_full_name}.check_names": "[\"openmetrics\"]"
+          "ad.datadoghq.com/${local.path_cleanup_full_name}.init_configs": "[{}]"
+          "ad.datadoghq.com/${local.path_cleanup_full_name}.instances": "[{ \"prometheus_url\": \"http://%%host%%:8008/actuator/prometheus\", \"namespace\": \"${local.instance_alias}\", \"metrics\": [\"path_cleanup_job_seconds_sum*\"] }]"
```

Comment:
my comment got lost. You need to use `var.k8s_namespace`

Reply:
No, this namespace isn't related to the `k8s_namespace`. It's just a prefix used to segregate the metrics.

Reply:
right okay.